Dockerfile.tests

```
@@ -1,6 +1,6 @@
 ARG LIBSLIRP_COMMIT=v4.3.1
 
-FROM ubuntu:18.04 AS build
+FROM ubuntu:20.04 AS build
 RUN apt update && apt install -y automake autotools-dev make gcc libglib2.0-dev libcap-dev libseccomp-dev git ninja-build python3-pip
```

```suggestion
RUN apt update && DEBIAN_FRONTEND=noninteractive apt install -y automake autotools-dev make gcc libglib2.0-dev libcap-dev libseccomp-dev git ninja-build python3-pip
```
(ref https://github.com/rootless-containers/slirp4netns/pull/233/files#r519023269)

---

And in the `test` container we need `ncat` explicitly.
```
RUN apt update && DEBIAN_FRONTEND=noninteractive apt install -y git libtool iproute2 clang clang-format clang-tidy iputils-ping iperf3 nmap ncat jq
```
(ref https://github.com/rootless-containers/slirp4netns/pull/233/commits/d6d7cc0014e47ef88cf6af9cd1b21eb388befb20 , https://github.com/rootless-containers/slirp4netns/pull/233#issuecomment-724175055 )
(GitHub doesn't allow me to make a suggestion for that since it's too far from current changes.)

---

thanks, added

---

Don't miss the `ncat` ;).
You probably also want something like https://github.com/rootless-containers/slirp4netns/pull/233/commits/a7d99afa92f86e5c5a89cf20afd5a32d739fc0c8 .

---

thanks, missed your ncat comment by race condition :)